src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -5717,10 +5717,10 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
     <value>Nullability of reference types in type of parameter doesn't match implemented member (possibly because of nullability attributes).</value>
   </data>
   <data name="WRN_UninitializedNonNullableField" xml:space="preserve">
-    <value>Non-nullable {0} '{1}' must contain a non-null value when exiting constructor. Consider declaring the {0} as nullable.</value>
+    <value>Non-nullable {0} '{1}' must contain a non-null value when exiting constructor. Consider adding the required modifier or declaring the {0} as nullable.</value>
```

Comment:
Can we change `required` to `'required'`?

Reply:
We aren't 100% consistent here. If you want to address this comment you could open a new PR, but it isn't necessary if you don't want to.

Reply:
Ok, I just going to use the same PR

Reply:
This pr is merged. You can't reuse it. You will need to open a new PR.

Reply:
Sorry, I meant the same issue, this is done on PR#72767